app/uk/gov/hmrc/apidocumentation/models/Documentation.scala

```diff
@@ -26,7 +26,7 @@ object APIAccessType extends Enumeration {
   val PRIVATE, PUBLIC = Value
 }
 
-case class APIAccess(`type`: APIAccessType.Value)
+case class APIAccess(`type`: APIAccessType.Value, isTrial: Option[Boolean] = None)
```

Comment:
This `isTrial` flag can have three states - is that intended?

Reply:
That's known and tested for. IsTrial may not be present and doesn't make sense for PUBLIC apis.